# List of points
points: [[0.0, 0.0, "N"], [1.0, 0.0,"N"], [1.0, 1.0, "N"], [0.0, 1.0, "N"], [0.0, 0.0, "N"]]
linear_vel_max: 1.0
angular_vel_max: 3.0
linear_vel_min: 0.05
angular_vel_min: 0.01
linear_velocity : 0.0
angular_velocity : 0.0